presto-cli/src/main/java/io/prestosql/cli/ClientOptions.java

```diff
@@ -125,6 +125,9 @@
     @Option(name = "--session", title = "session", description = "Session property (property can be used multiple times; format is key=value; use 'SHOW SESSION' to see available properties)")
     public final List<ClientSessionProperty> sessionProperties = new ArrayList<>();
 
+    @Option(name = "--extra-credential", title = "extra-credential", description = "Extra credentials (property can be used multiple times; format is key=value)")
```

Comment:
Can you please add this to documentation with the example how user can pass multiple extra credentials?

Also I don't like the name `extra-credential`. The `extra` says nothing about the meaning. Maybe it should be `connector-credential` or maybe just `credential`?